src/data/multiset/range.lean

```diff
@@ -33,4 +33,14 @@ theorem range_subset {m n : ℕ} : range m ⊆ range n ↔ m ≤ n := range_subs
 
 theorem self_mem_range_succ (n : ℕ) : n ∈ range (n + 1) := list.self_mem_range_succ n
 
+lemma range_add (a b : ℕ) :
+  range (a + b) = range a ∪ (range b).map (λ x, a + x) :=
```

Comment:
I would expect this statement to be expressed with `+`, which feels like a stronger statement. Perhaps both are worth having.
```